src/components/Forecast.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import cloudy from "../img/weather-icons/cloudy.svg";
+import drizzle from "../img/weather-icons/drizzle.svg";
+import fog from "../img/weather-icons/fog.svg";
+import partlycloudy from "../img/weather-icons/partlycloudy.svg";
+import rain from "../img/weather-icons/rain.svg";
+import storm from "../img/weather-icons/storm.svg";
+
+class Forecast extends React.Component {
+  state = {
+    city: "",
+    temp: "",
+    hours: "",
+    icons: ""
+  };
+  loadWeather = async e => {
+    e.preventDefault();
+
+    const api = await fetch();
+    const forcast = await api.json();
+    console.log(forcast);
```

Comment:
Don't fetch the weather again here. You should be receiving what you need via `props`, coming from the parent component `App`.